time stamp
---
nav_msgs/Odometry pos
bool stationary